apisix/utils/log-util.lua

```diff
@@ -82,23 +82,74 @@ local function gen_log_format(format)
     return log_format
 end
 
+local log_vars = {
+    latency = function() return (ngx_now() - ngx.req.start_time()) * 1000 end,
```

Comment:
The calculation logic for latency, upstream_latency, and apisix_latency has been reimplemented in log_vars. This is completely duplicated from the existing latency_details_in_ms function logic.